src/modules/LanItems.rb

```diff
@@ -2535,13 +2535,18 @@ def clear_set_hostname
     # @param [String] device type
```

Comment:
above for sentence `Raises an exception when type is incorrect.` correct yardoc is `@raise [ArgumentError] when type is nil or empty` . Not sure what it raise when type is unknown. So it should be documented.

Reply:
thanks